interfaces/seccomp/template.go

```diff
@@ -444,6 +444,7 @@ socket AF_LOCAL
 socket AF_INET
 socket AF_INET6
 socket AF_IPX
+socket AF_XDP
```

Comment:
XDP feels like a more privileged network protocol than others, however I see the base seccomp template already allows say AF_KEY which is also privileged (as is AF_PACKET etc) as we then assume AppArmor will confine this further. As such, whilst I would prefer this was not part of the base seccomp template, adding it here keeps things consistent.